packages/react/fast-components-react-base/src/select/select.tsx

```diff
@@ -158,7 +158,7 @@ class Select extends Foundation<SelectHandledProps, SelectUnhandledProps, Select
                 className={this.generateClassNames()}
                 onKeyDown={this.handleKeydown}
                 onClick={this.handleClick}
-                tabIndex={-1}
```

Comment:
won't this hide the element entirely from screen readers?

Reply:
No, the actual focusable element is the trigger button contained within the component.

Reply:
I don't think this is the correct usage of aria-hidden, MDN explicitly recommends against doing exactly this (focusable element as child of `aria-hidden="true"`): https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute

It's also not clear to me why this helps address focusability  of the element - what problem is adding this attribute solving?

Reply:
The point of the fix was to stop double narration but just removing the tab index seems to do it, that aria-hidden was redundant so removed it.